### DISCUSSION TITLE
ExpressLRS green solid light

### DISCUSSION THREAD

**Discussion by RODRIGOAGJ** [ExpressLRS/ExpressLRS#2460] [Q&A]
Hello everyone!
 I have a problem with my ELRS receiver RX, on my drone.  It has a green light in solid state.  It doesn't give me the Wi-fi option and my controller is not connected, it is even turned off.

  Any other wired option, or any method, to repair and return to the light blinking for the wifi 🛜 ExpressLRS of the RX receiver?

 Thank you

**Comment by gonzalo-bulnes** [comment]
Hi @RODRIGOAGJ,


Did you solve this issue?

If not: have you seen this question in the troubleshooting docs? Does it look like it could be what you observe?

https://www.expresslrs.org/quick-start/troubleshooting/#i-think-my-receiver-and-transmitter-are-bound-as-the-led-on-the-receiver-is-solid-but-i-do-not-have-stick-inputs-or-rssi-on-the-radio
